### PR TITLE
Feature: Created new home page

### PR DESCRIPTION
**Pull request by amnaadev**

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Beautiful Home Page</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+</head>
+<body>
+    <h1>Beautiful Home Page</h1>
```
Comment (sakydev) [inline home.html:10]: @amnaadev shouldn't this be a `h3` instead? What's the reason for `h1`?

File: home.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Beautiful Home Page</title>
```
Comment (sakydev) [inline home.html]:
```suggestion
    <title>My Grand Beautiful Home Page</title>
```

**Review by sakydev — CHANGES_REQUESTED** [review]
@amnaadev oveall looks good but we need to discuss a few things. Left some comments for you 😄

File: home.html (hunk 1 above)
Comment (amnaadev) [reply home.html:10]: because h1 is cooler. Don't mess with my code. Just go sleep.

**Review by sakydev — APPROVED** [review]
🎖️

**Pull request merged**